deploy/example_catalog/cr-app-datafabric610.json

```diff
@@ -0,0 +1,236 @@
+{
+  "apiVersion": "kubedirector.hpe.com/v1beta1",
+  "kind": "KubeDirectorApp",
+  "metadata": {
+    "name": "datafabric610"
```

Comment:
nit: since this is mostly for compute why did we change the name from mapr cluster to datafabric ?

Reply:
Since MapR started to change name to Datafabric, i thought of changing to datafabric only to match theirs

Reply:
Let's not give the name as datafabric since datafabric cluster is already there which also has db, streams and hive.  Calling this also datafabric can potentially cause confusion. Maybe we can call this dfyarn or dfhadoop

Reply:
i have renamed datafabric to datafabric compute and distro to dfcompute610